gluten-core/src/main/java/org/apache/gluten/substrait/expression/WindowFunctionNode.java

```diff
@@ -16,41 +16,50 @@
  */
 package org.apache.gluten.substrait.expression;
 
+import org.apache.gluten.expression.ExpressionConverter;
 import org.apache.gluten.substrait.type.TypeNode;
 
 import io.substrait.proto.Expression;
 import io.substrait.proto.FunctionArgument;
 import io.substrait.proto.FunctionOption;
 import io.substrait.proto.WindowType;
+import org.apache.spark.sql.catalyst.expressions.Attribute;
+import org.apache.spark.sql.catalyst.expressions.PreComputeRangeFrameBound;
 
 import java.io.Serializable;
 import java.util.ArrayList;
+import java.util.HashMap;
 import java.util.List;
 
+import scala.collection.JavaConverters;
+
 public class WindowFunctionNode implements Serializable {
   private final Integer functionId;
   private final List<ExpressionNode> expressionNodes = new ArrayList<>();
 
   private final String columnName;
   private final TypeNode outputTypeNode;
 
-  private final String upperBound;
+  private final org.apache.spark.sql.catalyst.expressions.Expression upperBound;
 
-  private final String lowerBound;
+  private final org.apache.spark.sql.catalyst.expressions.Expression lowerBound;
 
   private final String frameType;
 
   private final boolean ignoreNulls;
 
+  private final List<Attribute> originalInputAttributes;
+
   WindowFunctionNode(
       Integer functionId,
       List<ExpressionNode> expressionNodes,
       String columnName,
       TypeNode outputTypeNode,
-      String upperBound,
-      String lowerBound,
+      org.apache.spark.sql.catalyst.expressions.Expression upperBound,
+      org.apache.spark.sql.catalyst.expressions.Expression lowerBound,
```

Comment:
Can we import `org.apache.spark.sql.catalyst.expressions.Expression` and use it here?

Reply:
The `Expression` is conflict with `io.substrait.proto.Expression` which alreay imported in this class, and Java doen't have the alias grammer like scala